build/BuildSteps.cs

```diff
@@ -544,5 +552,52 @@ public static void AddGoZip()
                 }
             }
         }
+
+        public static void CreateIntegrationTestsBuildManifest()
+        {
+            if (!string.IsNullOrEmpty(Settings.IntegrationBuildNumber))
+            {
+                if (_manifest == null)
+                {
+                    // This should never be the case. The initialization takes place in UpdatePackageVersionsForIntegrationTests.
```

Comment:
Fixed. I've added a check in the branch. Here is what it will look like:
`if (!string.IsNullOrEmpty(Settings.IntegrationBuildNumber) && (_manifest != null)) { .. } `